receiver/mongodbatlasreceiver/README.md

```diff
@@ -8,7 +8,7 @@
 | Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Fmongodbatlas%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Areceiver%2Fmongodbatlas) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Fmongodbatlas%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Areceiver%2Fmongodbatlas) |
 | [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@schmikei](https://www.github.com/schmikei) \| Seeking more code owners! |
 
-[beta]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#beta
+[beta]: https://github.com/open-telemetry/opentelemetry-collector#beta
```

Comment:
Not super familiar but just following this link right now does not take me to appropriate beta description located here: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#beta

Reply:
@schmikei hmm, I don't think I changed this manually (just had it generated...). But I can double check.

Reply:
yeah...mdatagen makes the change you see here...